src/main/java/ratpack/zipkin/ServerTracingModule.java

```diff
@@ -47,7 +48,8 @@ protected void configure() {
     bind(ClientRequestAdapterFactory.class).in(SINGLETON);
     bind(ClientResponseAdapterFactory.class).in(SINGLETON);
 
-    bind(ZipkinHttpClient.class).to(ZipkinHttpClientImpl.class);
+
+    bind(HttpClient.class).annotatedWith(Zipkin.class).to(ZipkinHttpClientImpl.class);
```

Comment:
not 100% sure if I follow your question. if you are referring to `ZipkinHttpClientImpl`'s constructor `HttpClient` param, ratpack seems to provide a default `HttpClient` instance for that dependency